helm/defectdojo/templates/configmap.yaml

```diff
@@ -18,7 +18,8 @@ data:
   DD_CELERY_BROKER_SCHEME: {{ if eq .Values.celery.broker "redis" }}{{ template "redis.scheme" . }}{{ end }}
   DD_CELERY_BROKER_USER: ''
   DD_CELERY_BROKER_HOST: {{ if eq .Values.celery.broker "redis" }}{{ template "redis.hostname" . }}{{ end }}
-  DD_CELERY_BROKER_PORT: '{{ if eq .Values.celery.broker "redis" }}{{ .Values.redis.master.service.ports.redis | default "6379" }}{{ end }}'
+  DD_CELERY_BROKER_PORT: '{{ if eq .Values.celery.broker "redis" }}{{- if ( hasKey .Values.redis "master" ) -}}{{ .Values.redis.master.service.ports.redis }}{{ else }} "6379" {{ end }}{{- end -}}'
```

Comment:
Because this is already contained in quotes, providing "6379" as a raw value will result in unnecessary quoting (unlike the prior use of `default`). You can test [here](https://helm-playground.com/#t=LQhQFMA8EMFsAcA24BcACA3h4aB0A1aRAV3AGdcBLAF3FjLQB81qB7ATTkSbQDtLeAE3C9qaAExoAvlNAAROQH0AwgFEAMqoBK7RQCEtAeQDS2xQAVDWgCqLD6uegDkWNJQBmacAEc8hEuS4AMbgyABOAJ64AEZhrADW4GFoAERh4IKUZCnSUq4ERKQU6ZkUsNBktGG4ZEkAbpQhuPCsYdTFGVk8wu7QxIhiKQBsAMwA7ACcOTKuIoK5TvJKapo6%2BkamWhZWtgByqgDqzq4eXr4FARQh4VGxCUmpJVnTedhungAUaAAWFcbgET8hUCTwYKXKlSSOQAlGhgDMMEDLrhQbgIVUavVGuBmq12ijOgwEV5ELVcqlRpMcrMhLksDg5nCZIsgA&v=LQhQEsBcFMFsGcBcACA2gIgGbgE70ugDTLrzQDGA9gHYAm6AuqOdADbQ4CeioyyARjkoBrDinQ5otcPHSgAxMknSkCvslgBDfGLXrkZHADdwLHov18ADpRyRVFy32UyUARgBMAZgAsQA)

![Screenshot 2024-10-24 at 12 08 03](https://github.com/user-attachments/assets/d68cd924-c1ba-4545-99ff-8503098c9a7f)

```suggestion
  DD_CELERY_BROKER_PORT: '{{ if eq .Values.celery.broker "redis" }}{{- if ( hasKey .Values.redis "master" ) -}}{{ .Values.redis.master.service.ports.redis }}{{ else }}6379{{ end }}{{- end -}}'
```

Reply:
Wow, nice catch ! 
Thanks for the review. 
Corrected and pushed.